src/main/java/org/sonarlint/intellij/config/project/SonarLintProjectBindPanel.java

```diff
@@ -385,6 +421,7 @@ public void itemStateChanged(ItemEvent e) {
       searchProjectButton.setEnabled(bound);
       connectionComboBox.setEnabled(bound);
       configureConnectionButton.setEnabled(bound);
+      exportConfigurationButton.setEnabled(org.sonarlint.intellij.config.Settings.getSettingsFor(project).isBound() && bound);
```

Comment:
You can get rid of the long package name

Reply:
We have another Settings and Java does not allow import renaming using "as"